README.md

```diff
@@ -1567,120 +1573,19 @@ expected that subject: "filename?"        <1234789>
 </ex-because-1>
 
 <details>
-<summary>💬 Don't use it to augment boolean/null checks</summary>
-
-We think this function holds some misuse potential and therefore we would like to point out, 
-that `because` is not intended to augment boolean nor null checks. There are better instruments to fulfil the same.
-
-For instance, instead of checking on boolean properties/methods, see if there is not a predefined function 
-which already provides the same check + nice reporting with additional context. The reporting of predefined functions
-make it easier for you and others to understand the context of a failure. For instance, instead of:
-
-<ex-because-2>
-
-```kotlin
-expect(listOf(1, 2).isEmpty()).because("list should not be empty") {
-    toBe(true)
-}
-```
-↑ <sub>[Example](https://github.com/robstoll/atrium/tree/master/misc/tools/readme-examples/src/main/kotlin/readme/examples/ReadmeSpec.kt#L377)</sub> ↓ <sub>[Output](#ex-because-2)</sub>
-<a name="ex-because-2"></a>
-```text
-expected that subject: false
-◆ equals: true
-ℹ because: list should not be empty
-```
-</ex-because-2>
-
-Write
-
-<ex-because-3>
-
-```kotlin
-expect(listOf(1, 2)).isEmpty()
-```
-↑ <sub>[Example](https://github.com/robstoll/atrium/tree/master/misc/tools/readme-examples/src/main/kotlin/readme/examples/ReadmeSpec.kt#L382)</sub> ↓ <sub>[Output](#ex-because-3)</sub>
-<a name="ex-because-3"></a>
-```text
-expected that subject: [1, 2]        (java.util.Arrays.ArrayList <1234789>)
-◆ is: empty
-```
-</ex-because-3>
-
-The same applies to null checks. For instance, instead of checking for null for an absent key in a map
-
-<ex-because-4>
-
-```kotlin
-expect(mapOf("a" to 1)["a"]).because("key a should not be in the map") {
-    toBe(null)
-}
-```
-↑ <sub>[Example](https://github.com/robstoll/atrium/tree/master/misc/tools/readme-examples/src/main/kotlin/readme/examples/ReadmeSpec.kt#L386)</sub> ↓ <sub>[Output](#ex-because-4)</sub>
-<a name="ex-because-4"></a>
-```text
-expected that subject: 1        (kotlin.Int <1234789>)
-◆ equals: null
-ℹ because: key a should not be in the map
-```
-</ex-because-4>
-
-Write
-
-<ex-because-5>
-
-```kotlin
-expect(mapOf("a" to 1)).containsNotKey("a")
-```
-↑ <sub>[Example](https://github.com/robstoll/atrium/tree/master/misc/tools/readme-examples/src/main/kotlin/readme/examples/ReadmeSpec.kt#L391)</sub> ↓ <sub>[Output](#ex-because-5)</sub>
-<a name="ex-because-5"></a>
-```text
-expected that subject: {a=1}        (java.util.Collections.SingletonMap <1234789>)
-◆ does not contain key: "a"        <1234789>
-```
-</ex-because-5>
+<summary>💬 Use <code>because</code> only to give reasons for non-obvious assertions</summary>
 
-Or in case of a nullable property:
+`because` can be a useful tool for explaining why there is a certain assertion. Sometimes it is not directly obvious one
```

Comment:
Your PR will no already be in 0.16.0 thus I suggest you adopt to `expectation` already
```suggestion
`because` can be a useful tool for explaining why there is a certain expectation. Sometimes it is not directly obvious why one
```

Reply:
I’d prefer to make this change consistently throughout the project, if you don’t mind.

Reply:
```suggestion
`because` can be a useful tool for explaining why there is a certain assertion. Sometimes it is not directly obvious why one
```